javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,129 @@
+//flight booking full name
+console.log("flight booking full name function")
+function getFullname(FirstName,surName,gender,useFormalName){
+    if(gender === "male" && useFormalName === true){
+        console.log( `your custumer name lord ${FirstName} ${surName}`)
+    }
+        else if(gender === "female" && useFormalName === true){
+            console.log (`your custumer name lordes ${FirstName} ${surName}` ) 
+        }
+        else if(gender = "" && useFormalName ===false){
+            console.log(`your custemer name ${FirstName} ${surName}`)
+        }
+        else 
+           console.log(`enter information`)
+
```

Comment:
format code. currently, it makes it difficult to see opening and closing brackets.

Reply:
thank you naresh for you review